tools/shared.py

```diff
@@ -278,7 +278,11 @@ def env_with_node_in_path():
 
 
 def _get_node_version_pair(nodejs):
-  actual = utils.run_process(nodejs + ['--version'], stdout=PIPE).stdout.strip()
+  override = os.environ.get('OVERRIDE_NODE_JS_VERSION')
+  if override:
+    actual = override
+  else:
+    actual = utils.run_process(nodejs + ['--version'], stdout=PIPE).stdout.strip()
```

Comment:
Ideally this change would live only in test/ code and only effect `NODE_JS_TEST`, not the main version of node js that we use.

i.e. can you make local function in `test/common.py` that does this override?   Then use the local version instead of `shared.get_node_version`?